Apps/Common/src/AspNetConfiguration/StartupConfiguration.cs

```diff
@@ -434,8 +434,26 @@ public void ConfigureAuditServices(IServiceCollection services)
         public void ConfigureDatabaseServices(IServiceCollection services)
         {
             this.Logger.LogDebug("ConfigureDatabaseServices...");
-            services.AddDbContextPool<GatewayDbContext>(options => options.UseNpgsql(
-                    this.configuration.GetConnectionString("GatewayConnection")));
+            IConfigurationSection section = this.configuration.GetSection("Logging:SensitiveDataLogging");
+            bool isSensitiveDataLoggingEnabled = section.GetValue<bool>("Enabled", false);
+            this.Logger.LogDebug($"Sensitive Data Logging is enabled: {isSensitiveDataLoggingEnabled}");
+
+            services.AddDbContextPool<GatewayDbContext>(options => {
```

Comment:
Build is failing due to the curly brace not being on the next line.

Reply:
Fixed.